src/components/SearchBar.js

```diff
@@ -5,11 +5,13 @@ export default function SearchBar() {
   const [isExpanded, setIsExpanded] = useState(false);
   const [searchQuery, setSearchQuery] = useState('');
 
-  const expandSearch = () => {
+  const expandSearch = (e) => {
+    e.preventDefault();
     setIsExpanded(true);
   };
 
-  const collapseSearch = () => {
+  const collapseSearch = (e) => {
+    e.preventDefault();
```

Comment:
same [this](https://github.com/duchieupham/vietqr_web_2.0/pull/74/files#r1786594958)